src/common/cockpitwebserver.c

```diff
@@ -40,7 +40,7 @@
 gboolean cockpit_webserver_want_certificate = FALSE;
 
 guint cockpit_webserver_request_timeout = 30;
-gsize cockpit_webserver_request_maximum = 4096;
+gsize cockpit_webserver_request_maximum = 8192;
```

Comment:
Can we make this a `#define` please?  At the very least, `const`.

Reply:
Right, I made it a const.